pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -94,6 +94,9 @@
   hoc_overview_csc_header: 'Introducing CS Connections'
   hoc_overview_csc_desc: 'Our newest curriculum highlights the connections between computer science and other subjects like math, language arts, science, and social studies. Through these modules, projects, and lessons, classrooms can explore their usual subjects in exciting new ways!'
   hoc_overview_csc_learn_more: 'Learn more about CS Connections'
+  hoc_danceparty_alt_text: 'gif of characters dancing underneath the text, Code your own dance party'
+  hoc_ai_alt_text: 'Illustration of AI bot'
+  hoc_banner_alt_text: 'Hour of Code banner'
```

Comment:
Is this for the hoc banner for cs connections or for "more activities from partners"?

Reply:
banner!

Reply:
Cool - how about change this to: "The Hour of Code logo surrounded by a tiger, a smile emoji, a cupcake, an 8-bit blue worm, an octopus poetry instructor, and school subject icons representing math, language arts, biology, and art"

Reply:
oh oops I misunderstood your question - this is for the "more activities from partners" image, but I'll update the banner text with your suggestion!

Reply:
Notes that the CS connections banner is not currently translatable